tools/SqliteCompiler/Program.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SqliteCompiler
+{
+    public class Program
```

Comment:
I was just thinking the same thing. Without needing to detect architecture, this could easily be done in bash w/o the overhead of dotnet